container/dockershell.sh

```diff
@@ -1,32 +1,55 @@
-#!/bin/bash -x
+#!/bin/bash
 
 # path within container
 SETUP_PATH="/root/db2eventstore-IoT-Analytics/container/setup"
 USER_VOLUME=${HOME}/eventstore_demo_volume
 
+# Deployment type constants, default to cp4d
+declare -r deployTypeCp4d="cp4d"
+declare -r deployTypeDsx="dsx"
+declare -r deployTypeStandalone="standalone"
+declare -r defaultDeployType=$deployTypeCp4d
+
 if [ -f ${HOME}/.user_info ]; then
-    echo "File: '.user-info' found in the current directory."
-    echo "Extracting user information from the file."
+    printf "File: '.user-info' found in the current directory."
+    printf "Extracting user information from the file."
     EVENT_USER=$(grep  "username" ${HOME}/.user_info |awk {'print $2'})
     EVENT_PASSWORD=$(grep  "password" ${HOME}/.user_info |awk {'print $2'})
 fi
 
 function usage()
 {
 cat <<-USAGE #| fmt
+
 Description:
 This script is the entrypoint of the eventstore_demo container. The script takes
-target Event Store server's public IP, Watson Studio Local's username, and password.
-The script will start docker container in interactive mode using the 
-image: eventstore_demo:latest.
+target Event Store server's public IP, optionally the target Event Store server's REST
+endpoint (required if it differs from the public IP), optionally the target Event Store
+server's deployment type, and the deployment's user and password.
 
+The script will start docker container in interactive mode using the:
+image: eventstore_demo:latest.
 -----------
 Usage: $0 [OPTIONS] [arg]
 OPTIONS:
 ========
---IP        Public IP address of the target Event Store server.
---user      User name of the Watson Studio Local user
---password  Password of the Watson Studio Local user
+--IP        Public IP address or DNS name of the target Event Store server
+--IPR       The REST endpoint IP or DNS name of the Event Store server.
+            This is to be used when the REST endpoint differs from the Public IP
+            (i.e. --IP) ... typically the case for cp4d deployments
+--deploymentType
+            The deployment type of the Event Store server, valid options include (default is cp4d):
+               cp4d (Cloud Pak for Data deployments)
+               dsx (DSX/WSL deployments)
+               standalone (standalone/desktop container deployments)
```

Comment:
let's call this desktop or developer edition, as you could argue the WSL version is a standalone version (as in it does not require CP4D)

Reply:
Right !  That's part of the confusion, in the past we have referred to DSX/WSL as standalone.

If we do get a desktop addition at some point it'll likely be the same as the standalone/container addition - with that said I think using desktop now will cause confusion (as folks often ask if that's still supported).  Will think about this.